modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,138 @@
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in the NNCP or NAD file.
+* You created a NNCP CR and defined a customized `br-ex` bridge network configuration in the object configuration file.
+
+.Procedure
+
+. Choose one of the following methods:
++
+.. For OVS bonding, open your NMState configuration file in your editor and add the following configuration in the file:
++
+[source,yaml]
+----
+# ...
+interfaces:
+# ...
+- name: br-phy <1>
+  type: ovs-bridge
+  state: up
+  ipv4:
+    enabled: false
+    dhcp: false
+  ipv6:
+    enabled: false
+    dhcp: false
+  bridge:
+    allow-extra-patch-ports: true
+    port:
+    - name: patch-phy-to-ex
+    - name: ovs-bond
+      link-aggregation:
+        mode: balance-slb <2>
+        port:
+        - name: eno1
+        - name: eno2
+- name: patch-ex-to-phy <3>
+  type: ovs-interface
+  state: up
+  patch:
+    peer: patch-phy-to-ex
+- name: patch-phy-to-ex <4>
+  type: ovs-interface
+  state: up
+  patch:
+    peer: patch-ex-to-phy
+- name: eno1 <5>
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+- name: eno2 <5>
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+# ...
+----
+<1> Specify an additional bridge, such as `br-phy`, to manage custom network rules that the OVN-Kubernetes CNI adds to the network environments.
+<2> Enables `balance-slb` mode for the OVS bridge interface.
+<3> Specify the OVS virtual interface that uses an uplink switch to connect the additional bridge with the br-ex bridge.
+<4> Specify the OVS virtual interface that uses an uplink switch to connect the br-ex bridge with the additional bridge.
+<5> Specify the physical interfaces that exist on your primary network.
++
+.. For Kernel bonding,
+
+. Use the `cat` command to base64-encode the contents of the NMState configuration:
++
+[source,terminal]
+----
+$ cat <nmstate_configuration>.yaml | base64 <1>
+----
+<1> Replace `<nmstate_configuration>` with the name of your NMState resource YAML file.
+
+. For static IP address assignment, manually reboot each node that exists in your cluster. For DHCP address assignment, you can reboot all nodes in one reboot operation.
+
+. Optional: If after the reboot operation and you waited a few minutes, a node or some nodes do not have the `READY` status, enter the following commands:
++
+.. Check the current state of an Open vSwitch (OVS) configuration, especially details about bridges, ports, and interfaces, by entering the following command:
++
+[source,terminal]
+----
+$ ovs-vsctl show
+----
++ 
+.. If you identify the `Bridge br-ex` entry in the output, delete the entry by running the following command:
++
+[source,terminal]
+----
+$ ovs-vsctl del-br br-ex
+----
+
+. Use the `nmstate` utility tool to apply the NMState configuration to each node in your cluster:
++
+[source,terminal]
+----
+$ nmstatectl apply node.yml
```

Comment:
This should never be run directly by the user for any of the slb scenarios. I'll see if I can find an outline of the steps.

Reply:
Removed:

>  . Use the `nmstate` utility tool to apply the NMState configuration to each node in your cluster:
+
[source,terminal]
----
$ nmstatectl apply node.yml
----